docs/build/identity.rs/1.1/docs/faq.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: Frequently Asked Questions
+sidebar_label: FAQ
+description: Frequently Asked Question regarding IOTA Identity.
+image: /img/Identity_icon.png
+keywords:
+  - FAQ
+  - Frequently Asked Question
+  - Troubleshooting
+  - IOTA Identity
+---
+
+This page contains frequently asked questions regarding the Identity Library and Self Sovereign Identity in general.
+
+### What programming languages are supported by the IOTA Identity framework?
+
+We currently provide a Rust library and a JavaScript library for both the browser and Node.js via WebAssembly (Wasm) bindings. See the "Programming Languages" section for more information.
+
+### Do I need to have IOTA tokens to start building with IOTA Identity?
+
+At the moment you don't need IOTA tokens to create and manage identities, although we are exploring opportunities to utilize the token in the future.
+
+### How do I prove control over my DID?
+
+Control over an identity is ultimately tied to the control over cryptographic key material (something you have).
+
+### How do I store my private keys?
+
+Theoretically you can store the keys however you like. We provide a secure default using IOTA Stronghold where possible, which is a secure software implementation for isolating digital secrets with encrypted storage. For even better guarantees you could look into hardware based key storage.
+
+### Do I need a Permanode to use IOTA Identity?
+
+You can get started without one but currently you do require access to a Permanode (a node which stores the entire history of the Tangle) to reliably resolve identities in production. The trustworthiness of this node is very important, since a malicious node could respond with outdated identities, so ideally you should run one yourself or ensure the party supplying the node is trustworthy.
```

Comment:
```suggestion
You can get started without one, but currently, you require access to a Permanode (a node that stores the entire history of the Tangle) to reliably resolve identities in production. The trustworthiness of this node is very important since a malicious node could respond with outdated identities. So, ideally, you should run one yourself or ensure the party supplying the node is trustworthy.
```